src/drive/store/configureStore.js

```diff
@@ -35,7 +35,7 @@ const configureStore = options => {
     middlewares.push(createTrackerMiddleware())
   }
 
-  if (options.logger !== false) {
+  if (options.logger === true) {
```

Comment:
We should add a flag in order to be able to use it without having to change the code. WDYT?

Reply:
Sounds good! Something like 5ff40be03ca279d1b7c67346a3b12362fa1867d0 ?